db/data/20191029113201_backfill_note_on_already_checked_claims.rb

```diff
@@ -0,0 +1,21 @@
+require "csv"
+
+class BackfillNoteOnAlreadyCheckedClaims < ActiveRecord::Migration[5.2]
+  def up
+    return unless ENV["NOTES_TO_BACKFILL"]
+
+    rows = CSV.parse(ENV["NOTES_TO_BACKFILL"])
```

Comment:
Open question, I'm wondering if this should be JSON, rather than CSV. I'm unsure if we might have problems storing multiline strings as environment variables in the keyvault. At least with JSON it can be one line.

Reply:
It would be helpful if we could find some documentation on it, or play around somehow to confirm the behaviour. Because if we go with JSON, there's also the possibility that Azure will do some parsing of the newline control string `\n`. Maybe the safest option would be Base64?

Reply:
Actually I may have misunderstood what your concern is about the newlines. Is it that Azure may do something funny with them, or that they're hard to represent in a CSV?

Reply:
(Also, if the data given in the Trello is everything, then I wonder if it's moot; none of them seem to contain newlines?)

Reply:
Okay, I think I should stop talking. I thought you were talking about newlines in an individual note; you mean the newlines that form the structure of the CSV, right?

Reply:
Yeah, that's right. I bear many scars of trying to deal with CSVs cross-platform, and I'm not sure if Azure will like only Windows-style newlines (`\r\n`) or Unix-style newlines (`\n`) or both.